core/types/admin/types.go

```diff
@@ -39,6 +39,8 @@ type ValidatorInfo struct {
 	Power  int64          `json:"power"`
 }
 
+// type ValidatorInfo = types.Validator
+
```

Comment:
I need to sort through the duplicated types here an in core/types.  I don't recall why we have this core/types/admin package while many of the types exist in core/types already.